src/unix/fs.c

```diff
@@ -1243,8 +1243,17 @@ static int uv__fs_statx(int fd,
 
     no_statx = 1;
     return UV_ENOSYS;
+  } else if (rc == 1) { 
```

Comment:
I wonder if this condition should be folded with the one above. We could check `if (rc != 0)` and add a `errno != 0` to the list of checks. WDYT?

Reply:
Thank you, I have now merged with the parent `if block`.